content/install/installation_overview.md

```diff
@@ -52,10 +52,10 @@ services:
       - DRONE_SECRET=${DRONE_SECRET}
 ```
 
-Drone integrates with multiple version control providers, configured using environment variables. This example demonstrates basic GitHub integration.
+Drone does not have any builtin user management. Instead, authentication is done using OAuth and is delegated to one of multiple version control providers, configured using environment variables. The example above demonstrates basic GitHub integration.
 
 {{% alert %}}
-You must register Drone with GitHub to obtain the client and secret. The authorization callback url must match `<scheme>://<host>/authorize`
+You must [register Drone with GitHub](https://github.com/settings/applications/new) to obtain the `DRONE_GITHUB_CLIENT` and `DRONE_GITHUB_SECRET` used above. The authorization callback url must match `<scheme>://<host>/authorize`. 
```

Comment:
Could you update `<scheme>://<host>/` to `DRONE_HOST`?

You can mention that `DRONE_HOST` is `<scheme>://<host>/`, and provide an example if you would like.

Reply:
I'm also not sure we want to link to the `/settings/applications/net` url since many drone installations are github enterprise. Perhaps it would therefore be better to link to github application documentation instead.